main.tf

```diff
@@ -55,9 +51,4 @@ resource "aws_rds_cluster_instance" "aurora_db" {
   instance_class       = var.db_instance_class
 
   performance_insights_enabled = var.performance_insights_enabled
-
-  lifecycle {
-    prevent_destroy = false
```

Comment:
`prevent_destroy = false` is the default so no real point to keeping this around.